README.md

```diff
@@ -1,31 +1,23 @@
 # Build setup
 
-#### Windows note: use an Ubuntu VM
-
-Many Windows developers have found that setting up an Ubuntu virtual machine is less painful than getting Ruby and other prerequisites running on Windows.
-
-* Option A: Use [VMWare Player](https://my.vmware.com/web/vmware/free#desktop_end_user_computing/vmware_player/4_0) and an [Ubuntu 14.04 iso image](http://releases.ubuntu.com/14.04.1/ubuntu-14.04.1-desktop-amd64.iso)
-* Option B: Use vagrant ([install](https://docs.vagrantup.com/v2/installation/)):
-  1. First clone the code.org git repo to get the provided Vagrantfile (you will be able to skip step 1 of the common setup instructions): `git clone https://github.com/code-dot-org/code-dot-org.git`
-  1. `cd code-dot-org`
-  1. `vagrant up`
-  1. `vagrant ssh`
-  1. Goto step 2 of the common setup instructions
-* Option C: Use AWS EC2: [launch Ubuntu 14.04 AMI](https://console.aws.amazon.com/ec2/home?region=ap-northeast-1#launchAmi=ami-d9fdddd8)
+This document describes how to set up your workstation to develop for Code.org.  For additional "getting started" information, please see [Getting Started as a Developer at Code.org](https://docs.google.com/document/d/158MiIxJ_1JokzUkw43Wf2bFONPycYoCeXtKEU-ww0Xw/edit#heading=h.ya2d9ejyhjic).
```

Comment:
Since we have non-Code.org developers working with our open-source codebase, it might be worth leaving this out or make it explicit that this doc is only for internal employees